asset/src/data_generator/schema.ts

```diff
@@ -74,6 +78,18 @@ export default class Schema extends ConvictSchema<DataGenerator> {
                 default: false,
                 format: Boolean
             },
+            delay: {
+                doc: 'Time in seconds that a worker will complete a slice. Great'
```

Comment:
Maybe say time the worker waits after generating records before returning slice?